Explorer/Assets/Scripts/Utility/Storage/PersistentSetting.cs

```diff
@@ -106,6 +106,7 @@ public PersistentSetting(string key, T defaultValue)
         {
             this.key = key;
             this.defaultValue = defaultValue;
+            setValue!(key, defaultValue);
```

Comment:
What's the purpose of this set? By the flow the default value already should be taken

Reply:
Applied suggestion